docs/tutorial.md

```diff
@@ -85,6 +85,16 @@ adapter:
   search_query: repo:YOURUSERNAME/shepherd-demo path:/ filename:.eslintrc
 ```
 
+**Note:** you can provide an `org` instead of `search_query`. This will return a list of every visible repo in a github organization.
```

Comment:
👍 

Might be worth mentioning in the readme as well next to where `search_query` is currently mentioned.

Reply:
Might also be worth adding some context around _why_ you'd use `org` instead of `search_query` (if you're getting inconsistent search results or have more complicated filtering logic you'd like to run solely in `should_migrate`).